datarepoBootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -lt 7 ]]; then
+    echo "usage $0 <sam-url> <datarepo-id> <datarepo-ops-group-name> <ops-admin-email> <stewards-group-name> <steward-admin-email> <auth-token>"
+    exit 1
+fi
+
+SAM_URL=$1
+DATAREPO_ID=$2
+OPS_GROUP_NAME=$3
+OPS_ADMIN_EMAIL=$4
+STEWARDS_GROUP_NAME=$5
+STEWARD_ADMIN_EMAIL=$6
+AUTH_TOKEN=$7
+
+# Create the ops group
+curl -X POST --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}" 
+# Add the user as an admin of the ops group
+curl -X PUT --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}/admin/${OPS_ADMIN_EMAIL}" 
+
+# Get the email address for the ops group
+OPS_EMAIL=$(curl --header 'Content-Type: application/json' --header 'Accept: application/json' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}")
+
+# Create the stewards group
+curl -X POST --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${STEWARDS_GROUP_NAME}" 
```

Comment:
Why the context length 0 piece?

Reply:
curl wouldn't work for a POST with no data without it - even though swagger didn't need it

Reply:
i could pass -d {} instead of setting content length.